hedera-node/src/main/java/com/hedera/services/store/models/EvmTransaction.java

```diff
@@ -0,0 +1,88 @@
+package com.hedera.services.store.models;
+
+import com.hedera.services.context.TransactionContext;
+import com.hedera.services.context.properties.GlobalDynamicProperties;
+import com.hedera.services.store.contracts.stubs.StubbedBlockchain;
+import com.hederahashgraph.api.proto.java.AccountID;
+import com.hederahashgraph.api.proto.java.ContractCallTransactionBody;
+import com.swirlds.common.CommonUtils;
+import org.apache.tuweni.bytes.Bytes;
+import org.hyperledger.besu.crypto.SECPSignature;
+import org.hyperledger.besu.ethereum.chain.Blockchain;
+import org.hyperledger.besu.ethereum.core.Address;
+import org.hyperledger.besu.ethereum.core.Difficulty;
+import org.hyperledger.besu.ethereum.core.Hash;
+import org.hyperledger.besu.ethereum.core.ProcessableBlockHeader;
+import org.hyperledger.besu.ethereum.core.Transaction;
+import org.hyperledger.besu.ethereum.core.Wei;
+
+import java.util.Optional;
+
+import static com.hedera.services.exceptions.ValidationUtils.validateFalse;
+import static com.hedera.services.utils.EntityIdUtils.asSolidityAddressHex;
+import static com.hederahashgraph.api.proto.java.ResponseCodeEnum.CONTRACT_NEGATIVE_GAS;
+import static com.hederahashgraph.api.proto.java.ResponseCodeEnum.CONTRACT_NEGATIVE_VALUE;
+
+public class EvmTransaction {
+	private final GlobalDynamicProperties properties;
+	private final ContractCallTransactionBody transactionBody;
```

Comment:
In the model objects we are not assigning the protobufs. Generally we do not use the protobufs inside the model if the properties that we want to "assign" from them are small amount.